keras/backend/torch/random.py

```diff
@@ -208,8 +208,10 @@ def gamma(shape, alpha, dtype=None, seed=None):
 def binomial(shape, counts, probabilities, dtype=None, seed=None):
     dtype = dtype or floatx()
     dtype = to_torch_dtype(dtype)
-    counts = torch.ones(shape) * convert_to_tensor(counts)
-    probabilities = torch.ones(shape) * convert_to_tensor(probabilities)
+    counts = torch.ones(shape, device=get_device()) * convert_to_tensor(counts)
```

Comment:
Most likely we should not use `ones()` and instead use this https://pytorch.org/docs/stable/generated/torch.broadcast_to.html which will be more efficient. `counts = broadcast_to(counts, shape)`, etc.

Reply:
Thanks for the suggestion.  Makes sense. Done

Reply:
Running GPU Tests to verify they pass.

Reply:
All tests now pass (CPU and GPU)

Reply:
Yes you're right about that. I used broadcasting in tensorflow but `torch.ones` was being used by existing functions in the `torch/random.py` module so I decided to just go with it.